platform/api.htmlui/src/org/netbeans/spi/htmlui/HTMLViewerSpi.java

```diff
@@ -190,6 +192,15 @@ public ClassLoader getClassLoader() {
             return loader;
         }
 
+        /** List of resources available to the page.
+         *
+         * @return resources used in the page displayed by the HTML user interface
+         * @since 1.25
+         */
+        public URL[] getResources() {
```

Comment:
Are we sure we want `URL[]` and not `String[]`? Isn't the relative path to the main page important?

Reply:
Fixed.